lib/phoenix_html/form.ex

```diff
@@ -680,6 +683,16 @@ defmodule Phoenix.HTML.Form do
           <option value="user">User</option>
           </select>
 
+      select(form, :country, %{"Europe" => ["UK", "Sweden", "France"], …})
```

Comment:
Can we also support: `%{"Europe" => [{"UK", "uk}, ...]}` and `%{"Europe" => %{"UK" => "uk, ...}}`?